ipython_parallel/_version.py

```diff
@@ -1,2 +1,9 @@
-version_info = (4, 0, 0, 'dev')
+_version_major = 4
+_version_minor = 0
+_version_patch = 0
+_version_extra = 'dev'
```

Comment:
See what we did for nbconvert - left it basically the same but added _version_extra:

https://github.com/jupyter/jupyter_nbconvert/pull/1/files#diff-d7781741d938b7b8e52a1059d3772880